INSTALL.md

```diff
@@ -55,7 +55,7 @@ cmake -DFDEEP_BUILD_UNITTEST=ON ..
 make unittest
 cd ../..
 ```
-
+The Unit Tests require Python3 and Tensorflow should be installed as a Python Model. You could use *pip show tensorflow* in terminal or *import tensorflow* in Python to check your Tensorflow.
```

Comment:
"Tensorflow should be installed as a Python Model"
By "Model", do you mean "module"? Or maybe [package](https://stackoverflow.com/a/7948672/1866775)?